lib/ansible/modules/cloud/amazon/ec2_lc.py

```diff
@@ -147,7 +147,8 @@
 import traceback
 
 from ansible.module_utils.basic import *
-from ansible.module_utils.ec2 import *
+from ansible.module_utils.ec2 import ec2_argument_spec, ec2_connect, connect_to_aws, \
+    get_ec2_security_group_ids_from_names, get_aws_connection_info, AnsibleAWSError
```

Comment:
I prefer a second (and perhaps even third) `from ansible.module_utils.ec2 import` line to a line continuation (brackets also work, but are subjectively less elegant). Not a blocker though